data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,346 @@
+outfit "Unfettered Landing Permit"
+	"maintenance costs" 681
+	description "This item is needed in order to land on Unfettered planets, and its loss will cancel the arrangement."
+
+mission "Hai Legacy 3"
+	name "Unfettered Relic of old"
+	description "Beat the Lighting Bugs in a merciless fight. Careful, you only have one day."
+	landing
+	source "Firelode"
+	deadline 2
+	to offer
+		has "flagship model: Ancient Ladybug"
+	on offer
+		conversation
+			`As you land and get out of your ship you quickly notice you have attracted a lot of attention, probably due to this old design. One Unfettered that looks like an engineer comes to speak to you, warmed than usual. "Welcome. I do not know how you have obtained this ship, and some will probably see it as a sign. As you may know it is a relic of our past, a past we are quite proud of."`
+			`	"In truth, I do not personally care about all of that. All I care about is that this ship is old, and has technology that would be very valuable because of how rare it is. Thus, I challenge you for it!"`
+			choice
+				`	"What do I get if I win?"`
+					goto win
+				`	"Can't I just sell it to you?"`
+			`	"No, because in the eyes of my brethren that would mean I did not deserve this ship."`
+			label win
+			`	"You will win the right to access our worlds for just a meagre daily fee of 500 credits. You will not be allowed to land so easily as recently should you refuse this deal. Furthermore, this Ancient Ladybug may not be destroyed or sold."`
+			`	"Beware, for there are no rules, other than the fact you may only use one ship, and only one ship! Tell me then, what say you, will you take to the skies, or are you too effrayed?"`
+			choice
+				`	"I accept."`
+				`	"I have nothing to prove to you."`
+					decline
+			`	"Good! I will of course not be fighting you personally, as it is too risky to do so. I would also not want to deny my brethren the opportunity to prove themselves against a mighty human warrior!"`
+			`	You are not sure if they are being serious or not, but their friends seem to be in for it. What seems to be their leader approaches you. "So you think you deserve to fly this mighty ship of the Hai empire?"`
+			`	"Yeah, what makes you think that, human?" Another one adds.`
+			choice
+				`	(Say nothing.)`
+				`	"Are you trying to scare me out of the fight?"`
+					goto scare
+			`	"I guess we'll find out." Says a third.`
+			label scare
+			`	The leader gets closer and addresses you in a more serious tone, "You should be aware, <last>, this is not a friendly competition between allies. As far as I am concerned you are an enemy. You should see it more as a fight for you to truly earn this ship, and more, but make no mistakes: it has only one rule, and that is that nobody else than us may interfere."`
+			`	"Although sometimes the rules can bend a little. But you certainly did not earn that privilege."`
+				to display
+					"reputation: Hai (Unfettered)" < -1500
```

Comment:
It starts at -1000 and each sb captured downs it by 100

Reply:
added a comment here too, and it was changed to -2000 since to not punish players too much either
the idea here is that this fixes rep but if you been naughty it's harder to fix it, and ofc you have to pay for it